wp-content/themes/pawar2018/front-page.php

```diff
@@ -1,98 +1,88 @@
-<?php get_header(); ?>
+<?php
+/*
+Template name: Front Page Template
+ */
+get_header(); ?>
   <div class="promo-banner">
     <h1 class="banner-headline">
       Join Ameya at the June 15th Virtual Town Hall.
       <a href="http://watchparty.pawar2018.com" target="_blank">RSVP Now</a>
     </h1>
   </div>
   <main>
-    <section class="hero">
+    <section class="hero" style="background-image: url('<?php the_field('hero_image'); ?>');">
```

Comment:
Can probably remove [the legacy CSS that used to set the bg image](https://github.com/pawar-2018/pawar2018/blob/dev/wp-content/themes/pawar2018/sass/modules/_hero.scss#L2)?